app/src/main/java/io/github/djunicode/canteenapp/LoginActivity.java

```diff
@@ -90,7 +90,8 @@ public void onClick(View v) {
                 @Override
                 public void onClick(View view) {
                     emailString = email.getText().toString();
-                    passwordString = SHAHashing();
```

Comment:
Is hashing taken care of in some other way?

Reply:
I've commented it out temporarily for testing since it is a dummy account created by the web people. Hashed password sent wouldnt match